src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -31,25 +31,26 @@ public void isSamePerson() {
         // null -> returns false
         assertFalse(ALICE.isSamePerson(null));
 
-        // same name, all other attributes different -> returns true
+        // same name, all other attributes different -> returns false
```

Comment:
I think should add an additional test case here which is same phone number, all other attributes different, which returns true

Reply:
done